.github/pull_request_template.md

```diff
@@ -17,10 +17,9 @@ To test this backend PR you need to checkout the #XXX frontend PR.
 ## How to test:
```

Comment:
- did you mean to revert this file to an older version? Make sure to change it back to its current version if not.
- you can use Source Control in VS Code or `git status` to check which files have been updated before committing. Changes you're not expecting or that are not relevant to your branch can be discarded.

Reply:
I'm about to ask you about that, because when I use `git status`, it showed up that file which I don't even know what that, and I only added my controller file when I did `git add <controller name>`.

Reply:
If you see files that you did not update after using `git status` I would just discard those changes. Source Control in VSC makes that really easy if you have it set up.

For now, I would just copy/paste the code for this file from the dev branch and make another commit.